.github/workflows/ci.yml

```diff
@@ -27,6 +27,9 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
 
+      - name: Setup Bats and bats libs
+        uses: bats-core/bats-action@e4efaa4edd28f6c43ea5bd55c0a23bdd62360100 # 1.5.4
```

Comment:
```suggestion
        uses: bats-core/bats-action@f7defb94362f2053a3e73d13086a167448ea9133 # v1.10.0
```

Reply:
there is a newer release

Reply:
reverted, suggested change doesnt exist - https://github.com/bats-core/bats-action/tree/1.5.4